packages/turf/index.js

```diff
@@ -16,6 +16,7 @@ var turf = {
     within: require('@turf/within'),
     concave: require('@turf/concave'),
     difference: require('@turf/difference'),
+    dissolve: require('@turf/dissolve'),
```

Comment:
Might want to remove this until all of the TravisCI tests pass.

We will re-add this afterwards once everything is good to go.

https://travis-ci.org/Turfjs/turf/builds/194420002#L780